localstack/services/install.py

```diff
@@ -172,9 +172,11 @@ def install_kinesis_mock():
 
     machine = platform.machine().lower()
     system = platform.system().lower()
+    version = platform.version().lower()
 
     LOG.debug('getting kinesis-mock for %s %s', system, machine)
-    if machine == 'x86_64' or machine == 'amd64':
+    if ((machine == 'x86_64' or machine == 'amd64') and
+        "arm64" not in version and "arm32" not in version):
```

Comment:
we should stick consistently with single quotes for strings `'`

Reply:
This is actually an issue in the lint setup - I'll fix that in a follow-up PR, no need to change here.. 👍 (but thanks for the pointer!)